modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2271,9 +2271,23 @@ private void _insertIntoTable(
 				Column<?, ?> column = dynamicObjectDefinitionTable.getColumn(
 					objectField.getDBColumnName());
 
-				_setColumn(
-					preparedStatement, index++, column.getSQLType(),
-					values.get(objectField.getName()));
+				if (StringUtil.equals(
+						objectField.getBusinessType(),
+						ObjectFieldConstants.
+							BUSINESS_TYPE_MULTISELECT_PICKLIST)) {
+
+					String stringValue = String.valueOf(
+						values.get(objectField.getName()));
+
+					_setColumn(
+						preparedStatement, index++, column.getSQLType(),
+						stringValue.substring(1, stringValue.length() - 1));
```

Comment:
@p-albuquerque can we change to something like:

```
StringUtil.removeChars(stringValue, CharPool.OPEN_BRACKET, CharPool.CLOSE_BRACKET)
```
to make it more intuitive? or the key can have these characters?